### PR TITLE
#trivial - f-cookie-banner:  Update readme

### PR DESCRIPTION
**Pull request by AndyMayJE**

### CODE REVIEW

File: packages/components/organisms/f-cookie-banner/README.md [hunk 1]
```diff
@@ -131,6 +133,26 @@ Finally, use the generated bundle in your HTML page
...
+
+Props should be added to the [App.vue](https://github.com/justeat/fozzie-components/blob/master/packages/components/organisms/f-cookie-banner/f-cookie-banner-static/src/App.vue) Remember the `locale` prop will be replaced by the `build:static-files` task.
+
+In the code example below we add the `nameSuffix` prop:
+
+```html
+<template>
+    <cookie-banner
+        locale='da-DK'
+        should-absolute-position-reopen-link={false}
```
Comment (SachaZvetelman) [inline packages/components/organisms/f-cookie-banner/README.md]: Shouldn't this be `shouldAbsolutePositionReopenLink="false"`?

File: packages/components/organisms/f-cookie-banner/README.md [hunk 2]
```diff
@@ -131,6 +133,26 @@ Finally, use the generated bundle in your HTML page
...
+        nameSuffix="myName"
+    />
+</template>
+```
+
+Next run `yarn build` followed by `yarn build:static-files` The generated files will be added to the root `dist/static` folder.
```
Comment (SachaZvetelman) [inline packages/components/organisms/f-cookie-banner/README.md]:
```suggestion
Next run `yarn build` followed by `yarn build:static-files`. The generated files will be added to the root `dist/static` folder.
```

File: packages/components/organisms/f-cookie-banner/README.md (hunk 1 above)
Comment (AndyMayJE) [reply packages/components/organisms/f-cookie-banner/README.md]: Thanks, trying to write React again

Comment (SachaZvetelman) [reply packages/components/organisms/f-cookie-banner/README.md]: I'm glad you Reacted well to my comment 🙄 
Yeah, I'll see myself out now...

**Review by SachaZvetelman — DISMISSED** [review]

**Pull request closed without merging**